{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":9496,"level":50,"msg":"[CREATE-HISTORY-ERROR] DatabaseError [SequelizeDatabaseError]: Unknown column 'employee.employee' in 'where clause'\n    at Query.formatError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:223:5)\n    at addChunk (_stream_readable.js:309:12)\n    at readableAddChunk (_stream_readable.js:290:11)\n    at Socket.Readable.push (_stream_readable.js:224:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Unknown column 'employee.employee' in 'where clause'\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'employee.employee' in 'where clause'\",\n    sql: \"SELECT `employeeId`, `firstName`, `lastName`, `photos`, `bloodGroup`, `salary`, `hiredDate`, `deptId` FROM `employee` AS `employee` WHERE `employee`.`employee` = '1' LIMIT 1;\",\n    parameters: undefined\n  },\n  original: Error: Unknown column 'employee.employee' in 'where clause'\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'employee.employee' in 'where clause'\",\n    sql: \"SELECT `employeeId`, `firstName`, `lastName`, `photos`, `bloodGroup`, `salary`, `hiredDate`, `deptId` FROM `employee` AS `employee` WHERE `employee`.`employee` = '1' LIMIT 1;\",\n    parameters: undefined\n  },\n  sql: \"SELECT `employeeId`, `firstName`, `lastName`, `photos`, `bloodGroup`, `salary`, `hiredDate`, `deptId` FROM `employee` AS `employee` WHERE `employee`.`employee` = '1' LIMIT 1;\",\n  parameters: undefined\n}","time":"2020-03-14T04:52:38.430Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":8556,"level":50,"msg":"[CREATE-HISTORY-ERROR] ReferenceError: employeeId is not defined\n    at HistoryClass.createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\services\\history.services.js:14:31)\n    at createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\controller\\history.controller.js:4:39)\n    at C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\routes\\history.route.js:9:31\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Route.dispatch (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\index.js:281:22\n    at Function.process_params (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\index.js:275:10)\n    at Function.handle (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\index.js:174:3)\n    at router (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\index.js:47:12)\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\index.js:317:13)\n    at C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\index.js:284:7\n    at Function.process_params (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\index.js:275:10)\n    at Function.handle (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\index.js:174:3)\n    at router (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\index.js:47:12)\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\index.js:317:13)\n    at C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\express\\lib\\router\\index.js:284:7","time":"2020-03-14T05:00:03.619Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":8320,"level":50,"msg":"[CREATE-HISTORY-ERROR] TypeError: Cannot read property 'toString' of undefined\n    at HistoryClass.createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\services\\history.services.js:16:38)\n    at async createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\controller\\history.controller.js:4:24)","time":"2020-03-14T05:00:37.044Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":8320,"level":50,"msg":"[CREATE-HISTORY-ERROR] TypeError: Cannot read property 'toString' of undefined\n    at HistoryClass.createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\services\\history.services.js:16:38)\n    at async createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\controller\\history.controller.js:4:24)","time":"2020-03-14T05:01:55.578Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":4072,"level":50,"msg":"[CREATE-HISTORY-ERROR] TypeError: Cannot read property 'toString' of undefined\n    at HistoryClass.createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\services\\history.services.js:16:38)\n    at async createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\controller\\history.controller.js:4:24)","time":"2020-03-14T05:03:49.624Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":3900,"level":50,"msg":"[CREATE-HISTORY-ERROR] TypeError: date is not a function\n    at HistoryClass.createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\services\\history.services.js:26:26)\n    at async createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\controller\\history.controller.js:4:24)","time":"2020-03-14T05:13:26.537Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":11612,"level":50,"msg":"[CREATE-HISTORY-ERROR] DatabaseError [SequelizeDatabaseError]: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n    at Query.formatError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Execute.handler [as onResult] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:223:5)\n    at addChunk (_stream_readable.js:309:12)\n    at readableAddChunk (_stream_readable.js:290:11)\n    at Socket.Readable.push (_stream_readable.js:224:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [ '30fa1ab0-65b3-11ea-888f-9504b3d176b3', 'Invalid date', '1' ]\n  },\n  original: Error: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [ '30fa1ab0-65b3-11ea-888f-9504b3d176b3', 'Invalid date', '1' ]\n  },\n  sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n  parameters: [ '30fa1ab0-65b3-11ea-888f-9504b3d176b3', 'Invalid date', '1' ]\n}","time":"2020-03-14T05:18:08.411Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":9388,"level":50,"msg":"[CREATE-HISTORY-ERROR] TypeError: moment(...).toNow.tz is not a function\n    at HistoryClass.createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\services\\history.services.js:26:41)\n    at async createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\controller\\history.controller.js:4:24)","time":"2020-03-14T05:23:22.535Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":10440,"level":50,"msg":"[CREATE-HISTORY-ERROR] DatabaseError [SequelizeDatabaseError]: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n    at Query.formatError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Execute.handler [as onResult] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:223:5)\n    at addChunk (_stream_readable.js:309:12)\n    at readableAddChunk (_stream_readable.js:290:11)\n    at Socket.Readable.push (_stream_readable.js:224:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [ '622eed70-65b5-11ea-af0c-7f3a8714ef79', 'Invalid date', '1' ]\n  },\n  original: Error: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [ '622eed70-65b5-11ea-af0c-7f3a8714ef79', 'Invalid date', '1' ]\n  },\n  sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n  parameters: [ '622eed70-65b5-11ea-af0c-7f3a8714ef79', 'Invalid date', '1' ]\n}","time":"2020-03-14T05:33:49.957Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":13108,"level":50,"msg":"[CREATE-HISTORY-ERROR] DatabaseError [SequelizeDatabaseError]: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n    at Query.formatError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Execute.handler [as onResult] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:223:5)\n    at addChunk (_stream_readable.js:309:12)\n    at readableAddChunk (_stream_readable.js:290:11)\n    at Socket.Readable.push (_stream_readable.js:224:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [ '2df6f000-65b7-11ea-8288-8b1ed3a30057', 'Invalid date', '1' ]\n  },\n  original: Error: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [ '2df6f000-65b7-11ea-8288-8b1ed3a30057', 'Invalid date', '1' ]\n  },\n  sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n  parameters: [ '2df6f000-65b7-11ea-8288-8b1ed3a30057', 'Invalid date', '1' ]\n}","time":"2020-03-14T05:46:41.305Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":13036,"level":50,"msg":"[CREATE-HISTORY-ERROR] DatabaseError [SequelizeDatabaseError]: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n    at Query.formatError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Execute.handler [as onResult] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:223:5)\n    at addChunk (_stream_readable.js:309:12)\n    at readableAddChunk (_stream_readable.js:290:11)\n    at Socket.Readable.push (_stream_readable.js:224:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [ '8920f340-65b7-11ea-820f-e77f546bd7b1', 'Invalid date', '1' ]\n  },\n  original: Error: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [ '8920f340-65b7-11ea-820f-e77f546bd7b1', 'Invalid date', '1' ]\n  },\n  sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n  parameters: [ '8920f340-65b7-11ea-820f-e77f546bd7b1', 'Invalid date', '1' ]\n}","time":"2020-03-14T05:49:14.270Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":11480,"level":50,"msg":"[CREATE-HISTORY-ERROR] DatabaseError [SequelizeDatabaseError]: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n    at Query.formatError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Execute.handler [as onResult] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:223:5)\n    at addChunk (_stream_readable.js:309:12)\n    at readableAddChunk (_stream_readable.js:290:11)\n    at Socket.Readable.push (_stream_readable.js:224:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [ 'caef91a0-65b7-11ea-9ec6-5731ec587479', 'Invalid date', '1' ]\n  },\n  original: Error: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [ 'caef91a0-65b7-11ea-9ec6-5731ec587479', 'Invalid date', '1' ]\n  },\n  sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n  parameters: [ 'caef91a0-65b7-11ea-9ec6-5731ec587479', 'Invalid date', '1' ]\n}","time":"2020-03-14T05:51:04.654Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":9900,"level":50,"msg":"[NO-ROUTE] There is no route to /","time":"2020-03-14T05:59:37.674Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":9900,"level":50,"msg":"[GET-HISTORY-ATTENDACE-BY-ID-ERROR] Error: WHERE parameter \"employeeId\" has invalid \"undefined\" value\n    at MySQLQueryGenerator.whereItemQuery (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-generator.js:2184:13)\n    at C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-generator.js:2173:25\n    at Array.forEach (<anonymous>)\n    at MySQLQueryGenerator.whereItemsQuery (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-generator.js:2171:35)\n    at MySQLQueryGenerator.getWhereConditions (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-generator.js:2583:19)\n    at MySQLQueryGenerator.selectQuery (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-generator.js:1315:28)\n    at QueryInterface.select (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\query-interface.js:1122:27)\n    at C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\model.js:1757:34\n    at tryCatcher (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues [as _onImmediate] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (internal/timers.js:439:21)","time":"2020-03-14T05:59:40.566Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":7892,"level":50,"msg":"[DB-ERROR] ConnectionError [SequelizeConnectionError]: Unknown or incorrect time zone: 'utc'\n    at C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\connection-manager.js:133:19\n    at tryCatcher (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\promise.js:725:18)\n    at _drainQueueStep (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues [as _onImmediate] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (internal/timers.js:439:21) {\n  name: 'SequelizeConnectionError',\n  parent: Error: Unknown or incorrect time zone: 'utc'\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_UNKNOWN_TIME_ZONE',\n    errno: 1298,\n    sqlState: 'HY000',\n    sqlMessage: \"Unknown or incorrect time zone: 'utc'\"\n  },\n  original: Error: Unknown or incorrect time zone: 'utc'\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_UNKNOWN_TIME_ZONE',\n    errno: 1298,\n    sqlState: 'HY000',\n    sqlMessage: \"Unknown or incorrect time zone: 'utc'\"\n  }\n}","time":"2020-03-14T07:59:22.776Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":11508,"level":50,"msg":"[GET-HISTORY-ATTENDACE-BY-ID-ERROR] DatabaseError [SequelizeDatabaseError]: Table 'smm.history' doesn't exist\n    at Query.formatError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:223:5)\n    at addChunk (_stream_readable.js:309:12)\n    at readableAddChunk (_stream_readable.js:290:11)\n    at Socket.Readable.push (_stream_readable.js:224:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Table 'smm.history' doesn't exist\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_NO_SUCH_TABLE',\n    errno: 1146,\n    sqlState: '42S02',\n    sqlMessage: \"Table 'smm.history' doesn't exist\",\n    sql: 'SELECT * FROM HISTORY',\n    parameters: undefined\n  },\n  original: Error: Table 'smm.history' doesn't exist\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_NO_SUCH_TABLE',\n    errno: 1146,\n    sqlState: '42S02',\n    sqlMessage: \"Table 'smm.history' doesn't exist\",\n    sql: 'SELECT * FROM HISTORY',\n    parameters: undefined\n  },\n  sql: 'SELECT * FROM HISTORY',\n  parameters: undefined\n}","time":"2020-03-14T12:58:50.856Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":4088,"level":50,"msg":"[GET-HISTORY-ATTENDACE-BY-ID-ERROR] Error: WHERE parameter \"employeeId\" has invalid \"undefined\" value\n    at MySQLQueryGenerator.whereItemQuery (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-generator.js:2184:13)\n    at C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-generator.js:2173:25\n    at Array.forEach (<anonymous>)\n    at MySQLQueryGenerator.whereItemsQuery (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-generator.js:2171:35)\n    at MySQLQueryGenerator.getWhereConditions (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-generator.js:2583:19)\n    at MySQLQueryGenerator.selectQuery (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-generator.js:1315:28)\n    at QueryInterface.select (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\query-interface.js:1122:27)\n    at C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\model.js:1757:34\n    at tryCatcher (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues [as _onImmediate] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (internal/timers.js:439:21)","time":"2020-03-14T13:32:04.574Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":10344,"level":50,"msg":"[NO-ROUTE] There is no route to /history/1","time":"2020-03-14T14:19:43.235Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":10344,"level":50,"msg":"[CREATE-HISTORY-ERROR] TypeError: Cannot read property 'slice' of undefined\n    at HistoryClass.createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\services\\history.services.js:25:42)\n    at async createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\controller\\history.controller.js:4:24)","time":"2020-03-14T14:19:55.416Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":836,"level":50,"msg":"[CREATE-HISTORY-ERROR] DatabaseError [SequelizeDatabaseError]: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n    at Query.formatError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Execute.handler [as onResult] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:223:5)\n    at addChunk (_stream_readable.js:309:12)\n    at readableAddChunk (_stream_readable.js:290:11)\n    at Socket.Readable.push (_stream_readable.js:224:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`out`,`employeeId`) VALUES (?,?,?,?);',\n    parameters: [\n      'f9d1bf00-65ff-11ea-af4a-ef6e5bf01f7f',\n      'Invalid date',\n      null,\n      '1'\n    ]\n  },\n  original: Error: Incorrect datetime value: 'Invalid date' for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`out`,`employeeId`) VALUES (?,?,?,?);',\n    parameters: [\n      'f9d1bf00-65ff-11ea-af4a-ef6e5bf01f7f',\n      'Invalid date',\n      null,\n      '1'\n    ]\n  },\n  sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`out`,`employeeId`) VALUES (?,?,?,?);',\n  parameters: [ 'f9d1bf00-65ff-11ea-af4a-ef6e5bf01f7f', 'Invalid date', null, '1' ]\n}","time":"2020-03-14T14:27:47.109Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":14184,"level":50,"msg":"[CREATE-HISTORY-ERROR] ReferenceError: createData is not defined\n    at HistoryClass.createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\services\\history.services.js:26:29)\n    at async createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\controller\\history.controller.js:4:24)","time":"2020-03-14T14:28:18.491Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":8264,"level":50,"msg":"[CREATE-HISTORY-ERROR] ReferenceError: createHistory is not defined\n    at HistoryClass.createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\services\\history.services.js:26:29)\n    at async createHistory (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\source\\controller\\history.controller.js:4:24)","time":"2020-03-14T14:28:46.731Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":13360,"level":50,"msg":"[CREATE-HISTORY-ERROR] DatabaseError [SequelizeDatabaseError]: Data too long for column 'in' at row 1\n    at Query.formatError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Execute.handler [as onResult] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:223:5)\n    at addChunk (_stream_readable.js:309:12)\n    at readableAddChunk (_stream_readable.js:290:11)\n    at Socket.Readable.push (_stream_readable.js:224:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Data too long for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_DATA_TOO_LONG',\n    errno: 1406,\n    sqlState: '22001',\n    sqlMessage: \"Data too long for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [\n      'ac9bf3c0-6606-11ea-841b-5bb763e9e28c',\n      '3/14/2020, 10:15:41 PM',\n      '1'\n    ]\n  },\n  original: Error: Data too long for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_DATA_TOO_LONG',\n    errno: 1406,\n    sqlState: '22001',\n    sqlMessage: \"Data too long for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [\n      'ac9bf3c0-6606-11ea-841b-5bb763e9e28c',\n      '3/14/2020, 10:15:41 PM',\n      '1'\n    ]\n  },\n  sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n  parameters: [\n    'ac9bf3c0-6606-11ea-841b-5bb763e9e28c',\n    '3/14/2020, 10:15:41 PM',\n    '1'\n  ]\n}","time":"2020-03-14T15:15:44.035Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":7600,"level":50,"msg":"[NO-ROUTE] There is no route to /history","time":"2020-03-14T15:21:00.841Z","v":0}
{"name":"AplikasiPencatatPembelian","hostname":"DESKTOP-2138S88","pid":2172,"level":50,"msg":"[CREATE-HISTORY-ERROR] DatabaseError [SequelizeDatabaseError]: Data too long for column 'in' at row 1\n    at Query.formatError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Execute.handler [as onResult] (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:223:5)\n    at addChunk (_stream_readable.js:309:12)\n    at readableAddChunk (_stream_readable.js:290:11)\n    at Socket.Readable.push (_stream_readable.js:224:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Data too long for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_DATA_TOO_LONG',\n    errno: 1406,\n    sqlState: '22001',\n    sqlMessage: \"Data too long for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [\n      'a3f4b8f0-6607-11ea-8ded-81f2c4c7e82d',\n      'Sat Mar 14 2020 22:22:38 GMT+0700 (Indochina Time)',\n      '1'\n    ]\n  },\n  original: Error: Data too long for column 'in' at row 1\n      at Packet.asError (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\3D Objects\\ProjectSMM\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:223:5)\n      at addChunk (_stream_readable.js:309:12)\n      at readableAddChunk (_stream_readable.js:290:11)\n      at Socket.Readable.push (_stream_readable.js:224:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:181:23) {\n    code: 'ER_DATA_TOO_LONG',\n    errno: 1406,\n    sqlState: '22001',\n    sqlMessage: \"Data too long for column 'in' at row 1\",\n    sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n    parameters: [\n      'a3f4b8f0-6607-11ea-8ded-81f2c4c7e82d',\n      'Sat Mar 14 2020 22:22:38 GMT+0700 (Indochina Time)',\n      '1'\n    ]\n  },\n  sql: 'INSERT INTO `historyattendance` (`historyId`,`in`,`employeeId`) VALUES (?,?,?);',\n  parameters: [\n    'a3f4b8f0-6607-11ea-8ded-81f2c4c7e82d',\n    'Sat Mar 14 2020 22:22:38 GMT+0700 (Indochina Time)',\n    '1'\n  ]\n}","time":"2020-03-14T15:22:39.011Z","v":0}
